libs/wingsdk/src/std/resource.ts

```diff
@@ -167,12 +158,8 @@ export abstract class Resource extends Construct implements IResource {
     return App.of(scope).newAbstract(fqn, scope, id, ...props);
   }
 
-  /**
-   * @internal
-   * */
-  public _supportedOps(): string[] {
-    return [];
-  }
+  /** @internal */
+  public abstract get _liftMap(): LiftDepsMatrixRaw;
```

Comment:
Why does this have to be a getter?

```suggestion
  public readonly _liftMap?: LiftDepsMatrixRaw;
```

Maybe like this?

And then if not defined it defaults to `{}`... more robust

Reply:
Sure, I'm down to try this